Geolocator/Geolocator/Geolocator.Plugin.iOS/GeolocatorImplementation.cs

```diff
@@ -225,10 +229,23 @@ public Task<bool> StartListeningAsync(int minTime, double minDistance, bool incl
             if (this.isListening)
                 throw new InvalidOperationException("Already listening");
 
+			double desiredAccuracy = DesiredAccuracy;
+
+			// to use deferral, CLLocationManager.DistanceFilter must be set to CLLocationDistance.None, and CLLocationManager.DesiredAccuracy must be 
+			// either CLLocation.AccuracyBest or CLLocation.AccuracyBestForNavigation. deferral only available on iOS 6.0 and above.
+			if (this.energySettings != null && this.energySettings.DeferLocationUpdates && UIDevice.CurrentDevice.CheckSystemVersion (6, 0)) {
+				minDistance = CLLocationDistance.FilterNone;
+				desiredAccuracy = CLLocation.AccuracyBest;
+			}
+
             this.isListening = true;
-            this.manager.DesiredAccuracy = DesiredAccuracy;
+			this.manager.DesiredAccuracy = desiredAccuracy;
             this.manager.DistanceFilter = minDistance;
-            this.manager.StartUpdatingLocation();
+
+			if (this.energySettings != null && this.energySettings.ListenForSignificantChanges)
```

Comment:
?. usage would be good, but what happens if DeferLocationUpdate == false ? Then we may run into issues because the above is not set. These should just be one setting with it's default set to true.